hardware/ariadne/bootloaders/ariadne/main.c

```diff
@@ -135,6 +137,7 @@ int main(void)
 	}
 
 	/* Exit to user application */
+	wdt_reset(); // Just to give time for the user app to start
```

Comment:
I guess this should be a wdt_disable instead of wdt_reset, as you cannot be sure that the user will disable it. One more question, do you program the arduino locally or have you tested it in great distances too? The code is kinda fragile when there is big distance between you and the arduino being programmed.